File: mc/rmcp/src/main/java/calebxzhou/rdi/mc/common2/mcp/RMcpContainerPutData.java
package calebxzhou.rdi.mc.common2.mcp;

public record RMcpContainerPutData(
        boolean dryRun,
        int fromInventorySlot,
        int requestedCount,
        int movedCount,
        RMcpContainerData.Slot movedItem,
        RMcpContainerMoveData.Endpoint to
) {
}

File: mc/rmcp/common/src/main/kotlin/calebxzhou/rdi/mc/common2/mcp/RMcpContainerPutData.kt
package calebxzhou.rdi.mc.common2.mcp

@JvmRecord
data class RMcpContainerPutData(
    val dryRun: Boolean,
    val fromInventorySlot: Int,
    val requestedCount: Int,
    val movedCount: Int,
    val movedItem: RMcpContainerData.Slot,
    val to: RMcpContainerMoveData.Endpoint?
)
